contracts/committee/Committee.sol

```diff
@@ -226,6 +230,15 @@ contract Committee is ConsensusModule {
         _;
     }
 
+    modifier proposalIsAccepted() {
+        require(
+            positionCounts[proposal] >= quorum,
```

Comment:
the deadcode function `proposalAccepted` is wrong and superceded by the `committeeDecision` and the commit-reveal scheme.  My apologies.

It is ultimately the `consensus` contract that knows whether the proposal put before the committee is accepted; the `committee` just gets a `proposal` and takes a `committeeDecision` from which it doesnt know whether or not that means the proposal is accepted or not; that s for consensus to check.

so wrt the rewards, the committee members can simply claim their rewards through the committee contract; but committee itself can be blind to whether or not there was a reward for the proposal being accepted;

Reply:
so this should be removed, and also `proposalAccepted()` is wrong and should be removed